docs/rst/tutorials/core/discoveryserver/discoveryserver_complex.rst

```diff
@@ -0,0 +1,380 @@
+
+    .. code-block:: bash
+
+        export ROS_DISCOVERY_SERVER=<server_ip>:<port>"
+
+    Replace ``<server_ip>`` with the IP address of the Discovery Server and <port> with the connection port.
+
+2. ``FASTDDS_ENVIRONMENT_FILE``:
+
+    This variable allows you to specify the path to a JSON file that contains all the environment variables.
+    You can specify the IP addresses and ports of the Discovery Servers the clients should contact.
+    It provides more flexibility, as you can modify the locators (i.e., server addresses) dynamically at runtime.
+    To use this option, create a JSON file with the following structure:
+
+    .. code-block:: xml
+
+        {
+            "ROS_DISCOVERY_SERVER": "<server_ip>:<port>"
+        }
+
+    Save the file in a known location and set the environment variable to point to it:
+
+    .. code-block:: bash
+
+        export FASTDDS_ENVIRONMENT_FILE="<json_file_path>"
+
+Regardless of the environment variable used, make sure to set it in each of the containers where the ROS 2 nodes will be running.
+To facilitate the environment setup, and since more than one environment variable needs to be set, we will use the ``FASTDDS_ENVIRONMENT_FILE`` variable to include them all in a single JSON file.
+
+Since the Discovery Server, a *publisher* and a *listener* ``image_tools`` nodes are running on Host A, these two nodes within Host A's containers should be configured to point to the Wifi ip address ``192.168.1.165`` and to localhost.
+Additionally, we will configure all nodes to be |SUPER_CLIENT| and `Large Data Mode <https://fast-dds.docs.eprosima.com/en/latest/fastdds/use_cases/tcp/tcp_with_multicast_discovery.html>`__ as builtin transport.
+The json file for the nodes in **Host A** will look like:
+
+.. code-block:: xml
+
+    {
+        "ROS_DISCOVERY_SERVER": "TCPv4:[192.168.1.165]:42100",
+        "ROS_DISCOVERY_SERVER": "TCPv4:[127.0.0.1]:42101",
+        "ROS_SUPER_CLIENT": "TRUE",
+        "FASTDDS_BUILTIN_TRANSPORTS"="LARGE_DATA"
+    }
+
+On Host B, where other *publisher* and *subscriber* ``image_tools`` nodes will be running, the ``ROS_DISCOVERY_SERVER`` variable should point to the IP address of Host A on the Wifi network, as the Discovery Server is running on a different machine.
+The json file for the nodes in **Host B** will be:
+
+.. code-block:: xml
+
+    {
+        "ROS_DISCOVERY_SERVER": "TCPv4:[192.168.1.165]:42100",
+        "ROS_SUPER_CLIENT": "TRUE",
+        "FASTDDS_BUILTIN_TRANSPORTS"="LARGE_DATA"
+    }
+
+Finally, make the export of the ``FASTDDS_ENVIRONMENT_FILE`` in every node terminal, pointing to the JSON file path:
+
+.. code-block:: bash
+
+    export FASTDDS_ENVIRONMENT_FILE="<json_file_path>"
+
+Run ROS 2 demo nodes
+^^^^^^^^^^^^^^^^^^^^
+
+After all the configurations have been set, run the *publisher* and  *subscriber* client nodes in each host:
+
+.. tabs::
```

Comment:
I'd avoid using tabs. The tabs are for cases in which you can select between multiple options. In this case, you have to go over all options running each of them so there is no choice.